### ISSUE TITLE
Open-Pose joints are first 25 of 49  VIBE 3D joints

### ISSUE THREAD

**Issue by Haetsal-Lee** [skelemoa/quovadis#7]
I read that you used first 25 joints out of 49 joints. (from readme files  of Skeletics and Skeletics-mimetics)

But in the given script([/Scripts/parse_vibe_data.py](https://github.com/skelemoa/quovadis/blob/86bd7707ab54a2956f783ff6bd46aa4dc28126dc/Scripts/parse_vibe_data.py#L10)), you didn't used first 25 joints.

So, I want to recheck which joints did you used on the paper.

Thanks.

**Comment by Anirudh257** [comment]
@Haetsal-Lee In Skeletics, we have taken the common 17 joints from VIBE(Openpose) and Kinect 25 joints.

**Comment by Haetsal-Lee** [comment]
> @Haetsal-Lee In Skeletics, we have taken the common 17 joints from VIBE(Openpose) and Kinect 25 joints.

I am bit confused from what you wrote on README. 
```
VIBE pose estimate gives 49 joints in total out of which we are choosing the first 25 body joints common with Openpose. 
```

Are you saying that you used 17 joints? which are commons of VIBE and KinectV2?

**Comment by Anirudh257** [comment]
@Haetsal-Lee We performed experiments on VIBE extracted data from Skeletics-152. VIBE has 49 joints but the first 25 joints selected are the same as Openpose. 

We had also performed an experiment on the 17 joints that we found common between NTU-25 joints and Openpose-25 joints. Apologies for the confusion caused.
![image](https://user-images.githubusercontent.com/16001446/122452583-07215a80-cfc7-11eb-8fd7-4ccc8975dd2c.png)
